enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentProcessingService.java

```diff
@@ -410,14 +410,14 @@ private void processDocsSubmitResults(OwnerId ownerId, DocumentVerificationEntit
             docVerification.setErrorOrigin(ErrorOrigin.DOCUMENT_VERIFICATION);
             logger.info("Document verification ID: {}, failed: {}, {}",
                     docVerification.getId(), docSubmitResult.getErrorDetail(), ownerId);
-            auditService.audit(docVerification, "Document verification failed for user: {}", ownerId.getUserId());
+            auditService.audit(docVerification, "Document verification failed for user: {}, detail: {}", ownerId.getUserId(), docSubmitResult.getErrorDetail());
```

Comment:
Error detail is filled https://github.com/wultra/enrollment-server/pull/802/files#diff-26aa493e19b315afe12bf5109a4a6917933ba3f3e2539be648565116bdb18374R259 in case of `DocumentVerificationException` or `RemoteCommunicationException`.